io.botfather.Botfather.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: io.botfather.Botfather
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: botfather
+tags:
+  - proprietary
+finish-args:
+  - "--filesystem=home:rw" # let user use their scripts properly
```

Comment:
Shouldn't portal allow secure reads and writes?

Reply:
My understanding of portals was that I could use Qt methods like QFileDialog::getOpenFileName without having to worry about permissions at all.

I might got it wrong, but doing so didn't work for me. The QFileDialog doesn't allow the user to select anything within his home folder.

Furthermore the program can be scripted by the user to automatically generate images (not only screenshots) and save them using QImage::save. As far as I know QImage::save is not covered by portals, as it doesn't prompt the user.

Reply:
I took another look at it. For the time being the application requires rw permissions to the to work as intended. That might change in the future though.

Details: A script file tells the program how to automate something. Those script files often come with images and other data i.e. for image recognition purposes. While running a script such data is read by the program. By introducting a convenient way to store scripts in a central location we might get rit of the home rw permission eventually.